### DISCUSSION TITLE
#If what

### DISCUSSION THREAD

**Discussion by NurkaAmre** [community/community#34120] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello everyone, what happens if I upload **movies** to my GitHub and upload more and more **movies** 😸?

**Comment by radar-blip** [comment]
Eventually, you will run out of storage space. 😃

**Reply by NurkaAmre** [reply]
Oh, means there are limits?

**Reply by radar-blip** [reply]
Yes there are limits.

**Comment by airtower-luna** [comment, marked as answer]
Considering that files larger than 50 MB are blocked (except on LFS, where you'll quickly run out of free storage space and have to pay), you won't be able to upload most movies in the first place. :wink: 

See: https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github

**Reply by djsunaysunay** [reply]
[](url)

**Reply by NurkaAmre** [reply]
Thanks for the answer.
